source/user/howto/oidc-agent.rst

```diff
@@ -60,16 +60,16 @@ Next time you want to start oidc-agent from scratch, you will only have to do::
 
 * You should set OIDC_SOCK (this is not needed, if you did it before)::
 
-	$ eval (oidc-agent)
-        oidc-add deep-iam
+	$ eval $(oidc-agent)
+        $ oidc-add deep-iam
```

Comment:
Is the indenting corrent?

Reply:
$ eval $(oidc-agent)  is one line
$ oid-add deep-iam   is another

first '$' usually indicates "command line"
BTW if we could add e.g. https://pypi.org/project/sphinx_copybutton/ , i.e. that the fields can be copied in the buffer, we better remove first '$'. just a remark, not exactly related to the PR